src/applications/pensions/21p527ez/README.md

```diff
@@ -1,6 +1,96 @@
 # Form: 21P-527EZ - Application for Veterans Pension
 
-### Context
+## Routes (URLs)
+### Base URLs
+[https:www.staging.va.gov/pension/application/527EZ](https://staging.va.gov/pension/application/527EZ)
```

Comment:
Instead of `https:ww.va.gov`, should the URLs read `https://www.va.gov` (throughout)?

Reply:
Good catch. I will update.